### ISSUE TITLE
Integrate orb-starter-kit

### ISSUE THREAD

**Issue by zzamboni** [zzamboni/leanpub-orb#3]
I hadn't heard of the https://github.com/CircleCI-Public/orb-starter-kit when I first wrote this orb.  It might be nice to integrate it now.

**Comment by Lumexralph** [comment]
@zzamboni From what I understood reading the orb-starter-kit documentation, are you suggesting moving leanpub-orb to the structure of the kit?

**Comment by zzamboni** [comment]
@Lumexralph yes, that's the idea. I haven't read the orb-starter-kit docs thoroughly, so I don't know if it makes sense or how hard it would be, but feel free to take a stab if you'd like.

**Comment by Lumexralph** [comment]
@zzamboni I would love to try it out and get your feedback.

**Comment by Lumexralph** [comment]
@zzamboni Hi, I would require some tokens from you to be able to complete the integration, would you be kind to help out?

- A CircleCI Personal API Token (Must be Org admin to claim a namespace and publish production Orbs)

- A GitHub Personal Access Token (Requires Repo access and Admin Org if pushing to an Org that is not your personal  account)
